devDungeon/src/item/effects/SpeedEffect.java

```diff
@@ -10,7 +10,7 @@
  * application of the speed increase and its subsequent reversal.
  */
 public class SpeedEffect {
-  private static final EventScheduler EVENT_SCHEDULER = EventScheduler.getInstance();
+  private static final EventScheduler EVENT_SCHEDULER = new EventScheduler();
```

Comment:
Hmmm. Ich glaube, ich verstehe die Designentscheidung noch nicht. Damit würdest Du ja andauernd einen neuen `EventScheduler` anlegen - aber Du willst doch vermutlich immer auf den selben Scheduler zugreifen? 

Dazu kommt, dass der `EventScheduler` ein `System` ist und in `Game` verankert ist. Sollte der Zugriff dann nicht über `Game` kommen?

Reply:
Also der EventScheduler ist ja nur static, somit ist die Line komplett unnötig. Ich habe auch in den anderen Effekte diese Line gelöscht. Der Unterschied hier ist aber, dass unten in der Datei die Programmier-Aufgabe für die Speed Potion ist und ich wollte jetzt den EventScheduler noch als Hinweis behalten.

Reply:
ah, verstehe. hmmm. kriegst du das irgendwie anders hin, über `game.system(EventScheduler.class)` oder so? Das `new` ist wirklich ziemlich ... irreführend - und wenn die Studis das nachmachen, dann geht irgendwas in die Hose 😅

Reply:
Dann soll die Studis einfach `EventScheduler.scheduleAction` schreiben, habe jetzt einfach den Verweis auf die Klasse in der JavaDoc ergänzt